engine/dlib/src/dlib/dstrings.cpp

```diff
@@ -232,20 +232,22 @@ int dmStrCaseCmp(const char *s1, const char *s2)
 #endif
 }
 
-#if defined(ANDROID)
-    #if defined(__USE_GNU) && __ANDROID_API__ >= 23
-        #define DM_STRERROR_USE_GNU
-    #else
-        #define DM_STRERROR_USE_POSIX
-    #endif
-#elif defined(__EMSCRIPTEN__)
-    // Emscripten wraps strerror_r as strerror anyway
-    #define DM_STRERROR_USE_UNSAFE
-#else
-    #if (_POSIX_C_SOURCE >= 200112L || _XOPEN_SOURCE >= 600) && ! _GNU_SOURCE || defined(_WIN32) || defined(__MACH__)
-        #define DM_STRERROR_USE_POSIX
+#if !(defined(DM_STRERROR_USE_POSIX) || defined(DM_STRERROR_USE_GNU))
```

Comment:
Another example of where we wish to control this from the setup, rather than expose platform defines.